lib/liquid/autoescape/liquid_ext/block_body.rb

```diff
@@ -0,0 +1,47 @@
+require "liquid"
+require "liquid/autoescape"
+require "liquid/autoescape/template_variable"
+require "liquid/autoescape/safe_string"
+
+module Liquid
+  class BlockBody
```

Comment:
At first I was confused how all this works - why we're monkey-patching `BlockBody` instead of just `Variable`. It is explained in the [original pull request](https://github.com/restaumatic/liquid-autoescape/pull/2), but this required some Git-digging, having this in a comment here would be easier.

(note: this is probably also the motivation why we still maintain a fork despite the upstream library being active?)

Reply:
however the original gem did change to use `TemplateVariable` instead of monkey patching variable, so i'll try using it.